pkg/webhook/controlplane/ensurer.go

```diff
@@ -146,30 +160,20 @@ var (
 
 func ensureVolumeMounts(c *corev1.Container, version string) {
 	c.VolumeMounts = extensionswebhook.EnsureVolumeMountWithName(c.VolumeMounts, cloudProviderConfigKubeControllerManagerVolumeMount)
-
-	if mustMountEtcSSLFolder(version) {
-		c.VolumeMounts = extensionswebhook.EnsureVolumeMountWithName(c.VolumeMounts, etcSSLVolumeMount)
-	}
+	// Host certificates are mounted to accommodate OpenStack endpoints that might be served with a certificate
```

Comment:
This will add the volume mounts also to the `kube-apiserver` deployment.
I guess, this is intended, right? Can you then add it to the release note as well?